src/pages/available-equipment/AvailableEquipment.jsx

```diff
@@ -0,0 +1,14 @@
+import { Container, TitleText } from './AvailableEquipment.styles';
+import Header from '../../components/header/Header.jsx';
+import Footer from '../../components/footer/footer';
+import Table from './Table/InventoryTable.jsx';
```

Comment:
```suggestion
import InventoryTable from './Table/InventoryTable.jsx';
```

굳이 컴포넌트 명을 바꿔서 import하지 말아주세요. 헷갈리게 됩니다.